packages/dialog/dialog-wrapper.md

```diff
@@ -34,6 +34,7 @@ import { DialogWrapper } from '@spectrum-web-components/dialog';
         slot="click-content"
         headline="Dialog title"
         dismissable
+        dismiss-label="Close"
```

Comment:
are there other SWC components that expose multiple labels in their interface? If this is something that will not be needed in the long term with the rollout of localization in SWC, then I suggest we avoid introducing this change.

Reply:
@audknu I followed the existing pattern that's being used for DialogWrapper:

https://github.com/adobe/spectrum-web-components/blob/main/packages/dialog/src/DialogWrapper.ts#L47-L51